### ISSUE TITLE
[Bug report] Modified then eliminated

### ISSUE THREAD

**Issue by zAlweNy26** [SpriteOvO/Telegram-Anti-Revoke#66]
### Describe the bug
Overlaid text when both edited and deleted

### Steps to reproduce
1. Ask someone to modify a message
2. Ask them to delete it
3. Watch the result

### Expected behaviour
![Telegram_KXkWGE2E3c](https://user-images.githubusercontent.com/41266342/125952956-0ed24443-3642-43d8-81c6-1a0de104f663.png)

### Actual behaviour
The text of the message overlay the "eliminated edited" string like this :
![Telegram_KXkWGE2E6c](https://user-images.githubusercontent.com/41266342/125951274-c2006705-f3e5-4cb6-9a5f-45ed95bdc201.png)
"eliminato" means eliminated
"modificato" means edited
Example in italian, sorry.

### Configuration
* Operating system: Windows 10
* Version of Telegram: 2.8.8 x64
* Version of the plugin: 0.4.0

I suggest that when a modified message is deleted, the string "modified" is replaced with "eliminated" or just make it like the expected behaviour.

**Comment by SpriteOvO** [comment]
Thanks for reporting!

Displaying `edited` is the expected behavior, the solution to this bug is to change some UI value related to the width, I will try to fix it when I have time.

**Comment by zAlweNy26** [comment]
Is this the line of code that changes its length?

https://github.com/SpriteOvO/Telegram-Anti-Revoke/blob/d37790b13650b5055431160e724d2c04dc70fce5/Source/Core/IAntiRevoke.cpp#L246

**Comment by SpriteOvO** [comment]
> Is this the line of code that changes the width?
> 
> https://github.com/SpriteOvO/Telegram-Anti-Revoke/blob/d37790b13650b5055431160e724d2c04dc70fce5/Source/Core/IAntiRevoke.cpp#L246

No, it's not anywhere in this repository currently, to fix this bug would require reverse engineering to get a new width location and then writing code to modify it.

**Comment by zAlweNy26** [comment]
Oh ok, my bad, sorry !
